libraries/core_libs/consensus/src/dag/dag.cpp

```diff
@@ -542,6 +542,12 @@ uint DagManager::setDagBlockOrder(blk_hash_t const &new_anchor, uint64_t period,
     return 0;
   }
 
+  if (new_anchor == blk_hash_t(0)) {
```

Comment:
NULL_BLOCK_HASH is used everywhere, why not also here ?

Reply:
Fixed, thanks